multiclusterhub/charts/cluster-backup-chart/templates/oadp-operators-group.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright Contributors to the Open Cluster Management project
+
+# creates the OADP Operator namespace group
+# All operator resources will be installed under the {{ .Release.Namespace }} namespace
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: {{ .Values.oadpOperator.name }}-group
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-2"
+spec:
+  targetNamespaces:
+  - {{ .Release.Namespace }}
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822